### PR TITLE
Added 403 response

### PR DESCRIPTION
**Pull request by ksigWyatt**
**Max** added 403 response when when files other than ZIP are uploaded instead of displaying 200 HTTP response.

*Upon merge - close branch*

### CODE REVIEW

**Comment by ksigWyatt** [comment]
Re - Opened after changing base to *dev* and not *master*

**Comment by justin-vanwinkle** [comment]
@ksigWyatt @mXaln 
tRecorderApi/language.json is gitignored.  See if you can figure out how it made it into the commit.  I don't want in making it into the dev branch.  Thanks!

File: tRecorderApi/tRecorderApi/settings.py [hunk 1]
```diff
@@ -25,7 +25,7 @@
 # SECURITY WARNING: don't run with debug turned on in production!
```
Comment (ksigWyatt) [inline tRecorderApi/tRecorderApi/settings.py]: @justin-vanwinkle perhaps we ignore this file then?

**Comment by ksigWyatt** [comment]
@justin-vanwinkle I finished removing the language.json file

**Pull request merged**